services/datagrid-service.json

```diff
@@ -0,0 +1,442 @@
+                              "value": "REMOVE_ALL"
+                           }
+                        ],
+                        "image": "${IMAGE}",
+                        "livenessProbe": {
+                           "exec": {
+                              "command": [
+                                 "/opt/datagrid/bin/livenessProbe.sh"
+                              ]
+                           },
+                           "initialDelaySeconds": 60,
+                           "timeoutSeconds": 10,
+                           "periodSeconds": 20,
+                           "successThreshold": 1,
+                           "failureThreshold": 5
+                        },
+                        "readinessProbe": {
+                           "exec": {
+                              "command": [
+                                 "/opt/datagrid/bin/readinessProbe.sh"
+                              ]
+                           },
+                           "initialDelaySeconds": 60,
+                           "timeoutSeconds": 10,
+                           "periodSeconds": 20,
+                           "successThreshold": 2,
+                           "failureThreshold": 5
+                        },
+                        "name": "${APPLICATION_NAME}",
+                        "ports": [
+                           {
+                              "containerPort": 8080,
+                              "name": "http",
+                              "protocol": "TCP"
+                           },
+                           {
+                              "containerPort": 8888,
+                              "name": "ping",
+                              "protocol": "TCP"
+                           },
+                           {
+                              "containerPort": 11222,
+                              "name": "hotrod",
+                              "protocol": "TCP"
+                           }
+                        ],
+                        "volumeMounts": [
+                           {
+                              "name": "srv-data",
+                              "mountPath": "/opt/datagrid/standalone/data"
+                           },
+                           {
+                              "name": "srv-data",
+                              "mountPath": "/var/run/secrets/java.io/keystores"
+                           }
+                        ],
+                        "resources": {
+                           "requests": {
+                              "cpu": "0.5",
+                              "memory": "${TOTAL_CONTAINER_MEM}Mi"
+                           },
+                           "limits": {
+                              "memory": "${TOTAL_CONTAINER_MEM}Mi"
+                           }
+                        }
+                     }
+                  ],
+                  "initContainers": [
```

Comment:
Utilising a initContainer is no longer necessary and as the equivalent can be achieved by calling `createKeystoresFromSecrets` in `infinispan_service_profiles.sh`